### PR TITLE
Alignment fixes for spec- and sample data

### PR DESCRIPTION
**Pull request by javagl**
**tl;dr**: Some of the binary data in the `Specs/Data` and `Apps/SampleData` did not meet the alignment requirements that are stated in the tile formats specifications. This mainly refers to B3DM, I3DM and PNTS data. This is fixed with this PR.

---

When validating the Specs- and Sample data, the validator reported many issues of the form
```
          "type": "BINARY_INVALID_ALIGNMENT",
          "path": "content/1/0/1.b3dm",
          "message": "The byte length must be aligned to 8 bytes",
          "severity": "ERROR"
```

These are fixed with this PR, with the summaries:

#### Sample Data (27 files)
Before fix
- 22 files with errors
- 6 files with warnings
- 1 files with infos

After fix
- 9 files with errors
- 5 files with warnings
- 3 files with infos

#### Specs Data (246 files)
Before fix
- 62 files with errors
- 104 files with warnings
- 36 files with infos

After fix
- 57 files with errors
- 104 files with warnings
- 37 files with infos

---

Details:

The validation reports before and after the fix are attached here

[Cesium Specs and Sample data validation reports.zip](https://github.com/CesiumGS/cesium/files/10427006/Cesium.Specs.and.Sample.data.validation.reports.zip)

The affected files are
```
// Sample data
"Apps/SampleData/Cesium3DTiles/Classification/Photogrammetry/content.b3dm",
"Apps/SampleData/Cesium3DTiles/Hierarchy/BatchTableHierarchy/tile.b3dm",
"Apps/SampleData/Cesium3DTiles/Batched/BatchedColors/batchedColors.b3dm",
"Apps/SampleData/Cesium3DTiles/Instanced/InstancedOrientation/instancedOrientation.i3dm",
"Apps/SampleData/Cesium3DTiles/Instanced/InstancedWithBatchTable/instancedWithBatchTable.i3dm",
"Apps/SampleData/Cesium3DTiles/PointCloud/PointCloudConstantColor/pointCloudConstantColor.pnts",
"Apps/SampleData/Cesium3DTiles/PointCloud/PointCloudDraco/pointCloudDraco.pnts",
"Apps/SampleData/Cesium3DTiles/PointCloud/PointCloudBatched/pointCloudBatched.pnts",
"Apps/SampleData/Cesium3DTiles/PointCloud/PointCloudNormals/pointCloudNormals.pnts",
"Apps/SampleData/Cesium3DTiles/PointCloud/PointCloudRGB/pointCloudRGB.pnts",
"Apps/SampleData/Cesium3DTiles/PointCloud/PointCloudWithPerPointProperties/pointCloudWithPerPointProperties.pnts",
"Apps/SampleData/Cesium3DTiles/Tilesets/Tileset/ul.b3dm",
"Apps/SampleData/Cesium3DTiles/Tilesets/Tileset/ll.b3dm",
"Apps/SampleData/Cesium3DTiles/Tilesets/TilesetWithViewerRequestVolume/points.pnts",
"Apps/SampleData/Cesium3DTiles/Tilesets/TilesetWithViewerRequestVolume/ul.b3dm",
"Apps/SampleData/Cesium3DTiles/Tilesets/TilesetWithViewerRequestVolume/ll.b3dm",

// Specs data
//Specs/Data/Cesium3DTiles/Batched/BatchedDeprecated1/batchedDeprecated1.b3dm",
//Specs/Data/Cesium3DTiles/Batched/BatchedDeprecated2/batchedDeprecated2.b3dm",
"Specs/Data/Cesium3DTiles/Implicit/ImplicitTilesetWithJsonSubtree/content/1/0/1.b3dm",
"Specs/Data/Cesium3DTiles/Implicit/ImplicitTilesetWithJsonSubtree/content/1/0/0.b3dm",
"Specs/Data/Cesium3DTiles/Instanced/InstancedWithoutNormals/instancedWithoutNormals.i3dm",
"Specs/Data/Cesium3DTiles/Instanced/InstancedWithTransform/instancedWithTransform.i3dm",
"Specs/Data/Cesium3DTiles/Implicit/ImplicitTileset/content/0/0/0.b3dm",
"Specs/Data/Cesium3DTiles/Implicit/ImplicitTileset/content/1/0/0.b3dm",
"Specs/Data/Cesium3DTiles/Implicit/ImplicitTileset/content/1/0/1.b3dm",
"Specs/Data/Cesium3DTiles/Implicit/ImplicitTileset/content/1/1/0.b3dm",
"Specs/Data/Cesium3DTiles/Implicit/ImplicitTileset/content/1/1/1.b3dm",
"Specs/Data/Cesium3DTiles/Metadata/ImplicitSubtreeMetadata/content/1/0/1.b3dm",
"Specs/Data/Cesium3DTiles/Metadata/ImplicitSubtreeMetadata/content/1/0/0.b3dm"
```
(The "deprecated" ones have intentionally been omitted in the update!)

### CODE REVIEW

**Comment by ggetz** [comment]
Great! Thanks for the update @javagl!
